packages/ui/src/components/DatePicker/TimeInput.tsx

```diff
@@ -0,0 +1,29 @@
+import { Form, InputProps } from '../Form/Form'
+import React, { useState } from 'react'
+
+export interface TimeInputProps extends InputProps {
+	onTimeChange?: (value: string) => void
+}
+
+export function TimeInput({ placeholder, name, onTimeChange }: TimeInputProps) {
+	const [value, setValue] = useState('')
+
+	return (
+		<Form.Input
```

Comment:
kind of hacky to use the https://ariakit.org/ `Form` like this but it works! i think another way to have done something like the following (using a `<Form>` component with a ariakit form state)
https://github.com/highlight/highlight/blob/fb3b81e9988edbb3eda00f37b54ed9b900ed8602/frontend/src/pages/Auth/SignIn.tsx#L68-L120